### PR TITLE
Rust version 1.12

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

File: easybuild/easyconfigs/r/Rust/Rust-1.12.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'https://www.rust-lang.org'
+description = """Rust is a systems programming language that runs blazingly fast, prevents segfaults,
+ and guarantees thread safety."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://static.rust-lang.org/dist/']
+sources = ['rustc-%(version)s-src.tar.gz']
+
+builddependencies = [('Python', '2.7.11')]
```
Comment (boegel) [inline easybuild/easyconfigs/r/Rust/Rust-1.12.0-foss-2016b.eb]: @itkovian bump to .12

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2153.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/1460ab7e131f3ef4c821708e2daac1bc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6c3b3738800b13055e44fcfa83940b79 for a full test report.

**Comment by itkovian** [comment]
Apparently it also needs cmake.

**Comment by boegel** [comment]
@itkovian you'll need to add an easyconfig for CMake 3.6.2 with the `foss/2016b` toolchain:

```
EasyBuildError: 'Irresolvable dependencies encountered: CMake/3.6.2-foss-2016b'
```

**Comment by boegel** [comment]
@itkovian or just wait until the just opened #3682 gets merged :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2665.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0d97869074adfb6ee8835423092ab60f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2069.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/49873800e6b9b4d840ca1e32bd651afa for a full test report.

**Comment by itkovian** [comment]
Awesome. Progress by waiting. I can totally dig that.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @itkovian!

**Pull request merged**